packages/runtime/src/next/tests/server.api-handler-webhook.test.ts

```diff
@@ -109,7 +109,7 @@ describe('MakeswiftApiHandler', () => {
 
       // Assert
       expect(statusCode).toBe(200)
-      if (router === 'app' || true) {
+      if (router === 'app') {
```

Comment:
`revalidateTag` [should not be called](https://makeswifthq.slack.com/archives/C014SCS4G75/p1750678367865609?thread_ts=1750462168.060599&cid=C014SCS4G75) when running in the Pages Router.